### PR TITLE
Fixes windows build script

### PR DESCRIPTION
**Pull request by SeeYangZhi**
- bumped package versions, updated build scripts
  - locked protobuf>=3.12,<4 for requirement-hw.txt due to error as shown below:
windows_protobuf_error
- added GA workflow

### CODE REVIEW

File: .github/workflows/build_wine.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: Windows build CI Test
```
Comment (ivan-zynesis) [inline .github/workflows/build_wine.yml:1]:
```suggestion
name: Windows build and publish
```

File: contrib/build-linux/appimage/Dockerfile [hunk 2]
```diff
@@ -1,51 +1,61 @@
 # Note: we deliberately use an old Ubuntu LTS as base image.
 # from https://docs.appimage.org/introduction/concepts.html :
 # "[AppImages] should be built on the oldest possible system, allowing them to run on newer system[s]"
-FROM ubuntu:16.04@sha256:a4fc0c40360ff2224db3a483e5d80e9164fe3fdce2a8439d2686270643974632
+FROM ubuntu:18.04@sha256:9bc830af2bef73276515a29aa896eedfa7bdf4bdbc5c1063b4c457a4bbb8cd79
```
Comment (ivan-zynesis) [inline contrib/build-linux/appimage/Dockerfile:4]: Similar question, GA do not rely on this Dockerfile right?

**Comment by ivan-zynesis** [comment]
Skip `/contrib/deterministic-build` directory in the review, assuming they are duplicated as #11 , will review again after the base branch is merged.

File: contrib/build-linux/appimage/Dockerfile (hunk 2 above)
Comment (SeeYangZhi) [reply contrib/build-linux/appimage/Dockerfile:4]: Similar to #12, it is being used in the file [[contrib/build-linux/appimage/build.sh](https://github.com/BirthdayResearch/defichain-electrum/pull/13/files#diff-f8adb7985674339e5eafb97ac5414a113204a1b76e99a40aa10ba5e2e1755b83)].

**Comment by ivan-zynesis** [comment]
Superseded by #20

**Pull request closed without merging**